protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -389,13 +400,22 @@ mod test {
     use core::convert::TryInto;
 
     #[test]
-    fn test_check_flag() {
-        let protocol = crate::Protocol::MiningProtocol;
-        let flag_avaiable = 0b_0000_0000_0000_0000_0000_0000_0000_0000;
+    fn test_check_flags() {
+        let protocol = Protocol::MiningProtocol;
+        let flag_available = 0b_0000_0000_0000_0000_0000_0000_0000_0001;
```

Comment:
why you changed this test?

Reply:
thought of it as a way ensuring that check_flags works when both available and required flags are the same for MiningProtocol and JobDeclarationProtocol.

Reply:
@Fi3 when I did the revert I keep getting this error:
```
test setup_connection::test::test_check_flag ... FAILED
test setup_connection::test::test_has_work_selection ... ok
test setup_connection::test::test_set_requires_std_job ... ok

failures:

---- setup_connection::test::test_check_flag stdout ----
thread 'setup_connection::test::test_check_flag' panicked at v2/subprotocols/common-messages/src/setup_connection.rs:408:9:
assertion failed: SetupConnection::check_flags(protocol, flag_avaiable, flag_required)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    setup_connection::test::test_check_flag
    
    ```
    
    any idea on how I can go about it?

Reply:
@Fi3 I managed to fix the error, I think you can review again and give feedback